lib/matplotlib/axes/_axes.py

```diff
@@ -6147,12 +6147,9 @@ def _normalize_input(inp, ename='input'):
         if bins is None:
             bins = rcParams['hist.bins']
 
-        # xrange becomes range after 2to3
+        # Avoid shadowing the builtin.
```

Comment:
There's a lot of code between the argument definition and the renaming here; I think this really should be the first thing in the function to avoid any and all possible shadowing.